src/components/Disqus.vue

```diff
@@ -16,9 +16,12 @@ export default {
   },
 
   mounted() {
-    const { permalink } = this
+    const {
+      permalink,
+      $siteConfig: { url }
+    } = this
     window.disqus_config = function() {
-      this.page.url = permalink
+      this.page.url = `${url}${permalink}`
```

Comment:
maybe we can simply use `location.protocol + location.host+ location.pathname`?  `siteConfig.url` is not a required option.